explanations/function_evaluator.ipynb

```diff
@@ -0,0 +1,1046 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "plt.scatter(wealth_grid, vf_arr, alpha=0.5, zorder=-1, label=\"Pre-calculated values\")\n",
+    "plt.plot(\n",
+    "    wealth_grid,\n",
+    "    vf_arr,\n",
+    "    zorder=-1,\n",
+    "    label=\"Linear interpolation (matplotlib)\",\n",
+    ")\n",
+    "plt.scatter(wealth_grid, v_on_grid, color=\"orange\", s=7)\n",
+    "plt.scatter(\n",
+    "    new_wealth_values,\n",
+    "    v_on_new_values,\n",
+    "    color=\"orange\",\n",
+    "    s=35,\n",
+    "    label=\"Interpolated values\",\n",
+    ")\n",
+    "plt.xlabel(\"Wealth ($x$)\")\n",
+    "plt.ylabel(\"$V_T(x)$\")\n",
+    "plt.legend()\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Technical Details\n",
+    "\n",
+    "In the following, we will discuss the underlying functions that are used to implement\n",
+    "the function evaluator.\n",
+    "\n",
+    "### Label Translator\n",
+    "\n",
+    "The label translator is used to map the labels of dense discrete grids to their\n",
```

Comment:
But this will stay, no? So why discuss like this? Shouldn't it be:

> *(Once issue [#82](https://github.com/OpenSourceEconomics/lcm/issues/82) is tackled, )* LCM converts discrete grids into an internal grid that is directly usable as an index. Thus, the label translator simply is the identity function.